### PR TITLE
33 cleanup and tested session cookie

### PR DESCRIPTION
**Pull request by jcai28**

### CODE REVIEW

**Review by lemonshark05 — COMMENTED** [review]
I think setting the back-end URL directly in the front-end js is a bit like hard code. For the subsequent docker configuration, it is recommended to change it to "const backend = process.env.BACKEND_URL;" to get the URL from env.

**Comment by jcai28** [comment]
updated

File: frontend/src/services/service.js [hunk 1]
```diff
@@ -1 +1 @@
-export const backend = "http://localhost:8000";
\ No newline at end of file
+export const backend = process.env.BACKEND_URL
```
[the remark this answers is not in the document]
Comment (jcai28) [reply frontend/src/services/service.js]: no need to use dotenv, updated the env name and worked

**Review by JunjiaWangUSF — APPROVED** [review]
good job

**Review by lemonshark05 — COMMENTED** [review]
Good job

**Pull request merged**